.github/workflows/osrm-backend.yml

```diff
@@ -159,6 +159,49 @@ jobs:
           curl --retry-delay 3 --retry 10 --retry-all-errors "http://127.0.0.1:5000/route/v1/driving/13.388860,52.517037;13.385983,52.496891?steps=true"
           docker stop osrm-container
 
+  docker-alpine-image:
```

Comment:
Hey @knowname !

Thanks for this PR! If by any chance you have a bit more time it would be nice to also improve code here a bit: `docker-alpine-image` shares a lot of code with `docker-image` that's why it would be use smth like matrix strategy here to share CI code for different Docker base images. GH Docs: https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs (or there is example right in this file if you look how `build-test-publish` is implemented)

This PR is LGTM even in its current state and I am ready to merge that even now, but just asking if we can make it a bit better 😄